packages/transactions/src/coders/transaction.test.ts

```diff
@@ -15,27 +16,41 @@ describe('TransactionCoder', () => {
       maturity: 0,
       scriptLength: 0,
       scriptDataLength: 0,
-      inputsCount: 0,
+      inputsCount: 1,
       outputsCount: 0,
       witnessesCount: 0,
       receiptsRoot: B256,
       script: '0x',
       scriptData: '0x',
-      inputs: [],
+      inputs: [
+        {
+          type: InputType.Contract,
+          utxoID: { transactionId: B256, outputIndex: 0 },
+          balanceRoot: B256,
+          stateRoot: B256,
+          contractID: B256,
+          txPointer: {
+            blockHeight: 0,
+            txIndex: 0,
+          },
+        },
+      ],
       outputs: [],
       witnesses: [],
     };
 
     const encoded = hexlify(new TransactionCoder().encode(transaction));
 
     expect(encoded).toEqual(
-      '0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000d5579c46dfcc7f18207013e65b44e4cb4e2c2298f4ac457ba8f82743f31e930b'
+      '0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000d5579c46dfcc7f18207013e65b44e4cb4e2c2298f4ac457ba8f82743f31e930b0000000000000001d5579c46dfcc7f18207013e65b44e4cb4e2c2298f4ac457ba8f82743f31e930b0000000000000000d5579c46dfcc7f18207013e65b44e4cb4e2c2298f4ac457ba8f82743f31e930bd5579c46dfcc7f18207013e65b44e4cb4e2c2298f4ac457ba8f82743f31e930b00000000000000000000000000000000d5579c46dfcc7f18207013e65b44e4cb4e2c2298f4ac457ba8f82743f31e930b'
     );
 
     const [decoded, offset] = new TransactionCoder().decode(arrayify(encoded), 0);
 
     expect(offset).toEqual((encoded.length - 2) / 2);
-    expect(JSON.stringify(decoded)).toEqual(JSON.stringify(transaction));
+    expect(JSON.parse(JSON.stringify(decoded))).toMatchObject(
```

Comment:
the stringified json order was different, this is done to ignore the key-value ordering